B/Blosc2/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Blosc2"
+version = v"1.9.9"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/Blosc/c-blosc2.git", "f0a05c9df216d156104756ffbbdf7ba8e553eb2a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd c-blosc2/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DBUILD_EXAMPLES=OFF -DBUILD_STATIC=OFF ..
+make -j${nproc}
+make install
+install_license ../LICENSES/*.txt
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
```

Comment:
Can we see what are the problems with the other platforms

Reply:
> Can we see what are the problems with the other platforms

With Widows this issue https://github.com/Blosc/c-blosc2/issues/302

On other platforms, to be honest, I did not fully understand - something related to the vectorization instructions, or rather their absence. In fact, I started to deal with blosc2 and write a wrapper on Julia for it only yesterday. If you think that I was too hasty with this PR, then it is quite possible that you are right.

Reply:
For Windows, a patch like https://github.com/JuliaPackaging/Yggdrasil/blob/d936efb0feaa2665c1b25766d2a1bc2258738d6d/F/Fontconfig/bundled/patches/0002-fix-mkdir.mingw.patch should do it.

If you enable all supported platforms we can see the errors and look into what to do

Reply:
> For Windows, a patch like https://github.com/JuliaPackaging/Yggdrasil/blob/d936efb0feaa2665c1b25766d2a1bc2258738d6d/F/Fontconfig/bundled/patches/0002-fix-mkdir.mingw.patch should do it.

Perhaps it is better to wait until this PR  https://github.com/Blosc/c-blosc2/pull/303 is merged? In any case, this is still a candidate for release

> 
> If you enable all supported platforms we can see the errors and look into what to do

I did (I hope I did it right :))

Reply:
It looks like https://github.com/Blosc/c-blosc2/pull/303 was merged a few hours ago.